simple_toml_settings/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 from .settings import TOMLSettings
 
-__all__ = ["TOMLSettings"]
+__all__ = ["TOMLSettings", "xdg_config"]
```

Comment:
you don't need this change, the `xdg_config` module will not be exported. Also, this wouldn't acutally work since you need to import the file first before adding to `__all_`. 
```suggestion
__all__ = ["TOMLSettings"]
```